src/alchemtest/namd/access.py

```diff
@@ -30,3 +30,25 @@ def load_tyr2ala():
     return Bunch(data=data,
                  DESCR=fdescr)
 
+def load_idws():
+    """Load the NAMD IDWS dataset.
+
+    Returns
+    -------
+    data : Bunch
+        Dictionary-like object, the interesting attributes are:
+
+        - 'data' : the data files by alchemical leg
+        - 'DESCR': the full description of the dataset
+
+    """
+
+    module_path = dirname(__file__)
+    data = {'forward': glob(join(module_path, 'idws/*.fepout.bz2'))}
```

Comment:
If it's only a single file then you can be explicit
```suggestion
    data = {'forward': [join(module_path, 'idws', 'quickidws.fepout.bz2']}
```